packages/suite/src/storage/index.ts

```diff
@@ -5,7 +5,7 @@ import { reloadApp } from 'src/utils/suite/reload';
 import { migrate } from './migrations';
 import type { SuiteDBSchema } from './definitions';
 
-const VERSION = 48; // don't forget to add migration and CHANGELOG when changing versions!
+const VERSION = 49; // don't forget to add migration and CHANGELOG when changing versions!
```

Comment:
[changelog](https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/storage/CHANGELOG.md) is missing?